client/composeApp/src/commonMain/kotlin/screen/RankingScreen.kt

```diff
@@ -40,20 +43,31 @@ fun RankingScreen(
         }
     ) {
         Column(
-            modifier = Modifier.padding(it).padding(16.dp)
+            modifier = Modifier
```

Comment:
Column(
            modifier = Modifier
                .background(color = Color(0xFFF5F4A5))
                .padding(it)
                .padding(16.dp)
                .fillMaxSize()
                .verticalScroll(rememberScrollState())